buildspecs/signing.yml

```diff
@@ -7,28 +7,59 @@ version: 0.2
 # as an integration point. This does require the codebuild role to have proper permissions.
 # $PRIVATE_KEY_ARN is pulled from the environment as well. This is something that will probably
 # be specified by the CloudFormation template that this becomes a part of.
+# $ECS_AGENT_AMD_TAR and $ECS_AGENT_ARM_TAR get fed into this codebuild by codepipeline
 
 env:
-  variables:
-    ECS_AGENT_TAR: ecs-agent.tar
-    ECS_AGENT_TAR_SIGNATURE: ecs-agent.tar.asc
+  exported-variables:
+    - CODEBUILD_BUILD_ID
```

Comment:
Where do we use CODEBUILD_BUILD_ID? Or will we use it in later development?

Reply:
Yep, that's the thing that I'm going to use to take the logs from Cloudwatch logs and put the things in the S3 bucket.